src/main/java/apoc/schema/Schemas.java

```diff
@@ -36,6 +36,23 @@
     @Context
     public KernelTransaction tx;
 
+    @Procedure(value = "apoc.schema.recreate", mode = Mode.SCHEMA)
+    @Description("apoc.schema.recreate() - recreates indexes and constraints")
+    public Stream<AssertSchemaResult> schemaRecreate(@Name(value = "config", defaultValue = "{}") Map<String, Object> config) throws ExecutionException, InterruptedException, TimeoutException {
+        long allIndexesTimeout = (long) config.getOrDefault("retrieveIndexesTimeout", 10000L);
+        long dropAllIndexesTimeout =  (long) config.getOrDefault("dropAllIndexesTimeout", 30000L);
+        long createAllIndexesTimeout = (long) config.getOrDefault("createAllIndexesTimeout", 30000L);
+
+        long allConstraintsTimeout = (long) config.getOrDefault("retrieveConstraintsTimeout", 10000L);
+        long dropAllConstraintsTimeout =  (long) config.getOrDefault("dropAllConstraintsTimeout", 30000L);
+        long createAllConstraintsTimeout = (long) config.getOrDefault("createAllConstraintsTimeout", 30000L);
```

Comment:
we can just have a config object instead of individual timeout

Reply:
I'm not sure what a config object is - could you elaborate?

Reply:
just a object that has all those timeouts (encapsulate parameters)

Reply:
just a regular java object that we can pass to the methods.